kubernetes.go

```diff
@@ -4,6 +4,7 @@ import (
 	"bytes"
 	"context"
 	"encoding"
+	"encoding/json"
```

Comment:
Why do we need this change?

Reply:
We use it for the MarshalJSON and UnmarshalJSON methods in this file. This allows us to marshal/unmarshal the integer values into a human readable form.

Reply:
I saw that. I was just surprised that we didn't already have a way to do that. But I see @bouk recommended it, so I'll defer to him. 😄